compiler/rustc_feature/src/builtin_attrs.rs

```diff
@@ -875,6 +875,11 @@ pub const BUILTIN_ATTRIBUTES: &[BuiltinAttribute] = &[
         lang, Normal, template!(NameValueStr: "name"), DuplicatesOk, EncodeCrossCrate::No, lang_items,
         "lang items are subject to change",
     ),
+    rustc_attr!(
+        rustc_as_ptr, Normal, template!(Word), ErrorFollowing,
+        EncodeCrossCrate::Yes,
+        "#[rustc_as_ptr] is used to mark functions that implement `as_ptr` or `as_ptr_mut`."
```

Comment:
Well, "implement" is usually refers to traits in Rust. I guess maybe it should say something about providing a pointer to the inner allocation instead.

Reply:
Good catch, that could end up being confusing. Just fixed